Plugins/DD4hep/src/ConvertDD4hepDetector.cpp

```diff
@@ -167,8 +167,8 @@ std::shared_ptr<const CylinderVolumeBuilder> volumeBuilder_dd4hep(
                << (subDetector.type() == "compound" ? "yes" : "no"));
 
   if (subDetector.type() == "compound") {
-    ACTS_VERBOSE("Subdetector : '" << subDetector.name()
-                                   << "' has type compound ");
+    ACTS_VERBOSE("Subdetector: '" << subDetector.name()
+                                  << "' has type compound ");
```

Comment:
I think there are at least two more cases of ` :` in the file. Should we change this consistently?

Reply:
they somehow slipped through the search